tools/lint-includes

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+# Copyright 2020 Redpanda Data, Inc.
```

Comment:
😄 Insert bad joke about getting used to writing the new year

Reply:
Yeh. Git knows the real date